src/index.js

```diff
@@ -47,24 +45,31 @@ module.exports = (
     debugLog = false
   } = {}
 ) => {
+  const processedEntry = Object.create(null);
+  if (entry instanceof Array) {
+    entry.forEach(file => {
+      processedEntry[basename(file)] = pathResolve(file);
+    });
+  }
+  else if (typeof entry === 'object') {
+    Object.keys(entry).forEach(item => {
+      if (!item.endsWith('.js') && !item.endsWith('.mjs'))
+        processedEntry[item + '.js'] = pathResolve(entry[item]);
+      else
+        processedEntry[item] = pathResolve(entry[item]);
+    });
+  }
+  else if (typeof entry === 'string') {
+    processedEntry['index.js'] = pathResolve(entry);
```

Comment:
```suggestion
    processedEntry[entry] = pathResolve(entry);
```

Reply:
We can change the default to the basename, but the default filename for any string value input right now is `index.js`.